### DISCUSSION TITLE
Можно ли настроить zapret на vps?

### DISCUSSION THREAD

**Discussion by simpmeth** [bol-van/zapret#1002] [Q&A]
есть vps c ubuntu , на нем развернут openvpn, можно юзать zapret для ускорения?

**Comment by bol-van** [comment]
zapret ничего не ускоряет. На VPS он и имеет смысл только, если сам VPS находится в зацензурированной стране. Например, в RU

**Comment by ygguser** [comment]
>есть vps c ubuntu , на нем развернут openvpn, можно юзать zapret для ускорения?

У меня установлен zapret на VPS, с размещением в России.
Работает. YouTube без рекламы.
Перед запуском zapret обязательно установите и настройте на сервере dnscrypt-proxy, т.к., в России провайдерами подменяются DNS-запросы и результаты работы `blockcheck.sh` могут оказаться некорректными.
